vortexor/Cargo.toml

```diff
@@ -0,0 +1,61 @@
+[package]
+name = "solana-vortexor"
+description = "Solana TPU Vortexor"
+documentation = "https://docs.rs/solana-vortexor"
+default-run = "solana-vortexor"
+publish = false
```

Comment:
I see the crate has `publish = false` set now, but from what I can tell, it looks like this would be a separate service run on a separate bin. Should this live in a different repo ?